config/routes.rb

```diff
@@ -29,6 +29,9 @@
   get "/login", to: 'sessions#new'
   post "/login", to: 'sessions#create'
 
+  get "/logout", to: 'sessions#destroy'
+  delete "/logout", to: 'sessions#destroy'
+
```

Comment:
I think we might be able to handle this with one route using a 'patch' verb going to sessions controller